lib/pymedphys/docs/lib/howto/get-started.rst

```diff
@@ -33,31 +73,31 @@ following in a terminal or command prompt:
 
     pip install pymedphys[user]
 
+This command can be copied into a command prompt on Windows with a "Right Click"
+of the mouse.
+
 You may need to open and close your terminal if you have only just installed
 Python. The ``[user]`` option is needed to install pymedphys with its
 "batteries included" so-to-speak. It will go and install a range of
 dependencies which you may need during your use of pymedphys.
 
+SSL Issues
```

Comment:
Within our network, we not only have SSL issues but also need to specify a proxy server for pip to use (I'm guessing this is common on health networks). This can be done via the command line, but hard to provide specific guidance because the user would need to know the proxy server in their network. Best to just leave it out here? Or if you like I could prepare a snippet to put in here with the command and explaining they need to find out the proxy server.

Reply:
> Or if you like I could prepare a snippet to put in here with the command and explaining they need to find out the proxy server.

@pchlap that would be amazing. Having an example here that someone can use to fill in the blanks for their own system would be significantly better than having no guidance at all.

Reply:
Will do, I'll add it directly to the branch of this pull request? Or rather a fresh branch?

Reply:
If it's just a small addition happy to have it included directly into this PR. But really whichever you prefer.

Reply:
Cool, I've popped something into this PR. Hopefully that's suitable?